validator/register.js

```diff
@@ -0,0 +1,56 @@
+const Validator = require('validator');
+const isEmpty = require('./is-empty');
+
+module.exports = function validateRegisterInput(data) {
+  let errors = {};
+
+  data.name = !isEmpty(data.name) ? data.name : '';
+  data.lastname = !isEmpty(data.lastname) ? data.lastname : '';
+  data.email = !isEmpty(data.email) ? data.email : '';
+  data.password = !isEmpty(data.password) ? data.password : '';
+  data.confirmPassword = !isEmpty(data.confirmPassword) ? data.confirmPassword : '';
+
+  if (!Validator.isLength(data.name, { min: 2, max: 30 })) {
```

Comment:
Instead of adding a dozen if statements, why not add a for loop and one if statement? Also, why are you checking twice if a field is empty or not? There will be an efficient way of doing this.